### PR TITLE
Add demo REST controller

### PR DESCRIPTION
**Pull request by Sephsekla**
## What?

This PR adds a demo REST Controller for fetching notifications.

## Why?

This allows us to move the demo content to the PHP side, allowing real js querying of the endpoint.

### CODE REVIEW

File: includes/restapi/class-notification-controller.php [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+		register_rest_route(
+			self::NAMESPACE,
+			self::NOTIFICATION_BASE,
+			array(
+				'methods'             => WP_REST_Server::READABLE,
+				'args'                => array(),
+				'callback'            => array( $this, 'get_notifications' ),
+				'permission_callback' => array( $this, 'authenticate' ),
+			),
+			false,
```
Comment (erikyo) [inline includes/restapi/class-notification-controller.php]:
```suggestion
			false
```
@Sephsekla if you could remove this comma that causes the failing action, we can merge this commit, so I start the rebase with the js branch with also this commit

**Comment by Sephsekla** [comment]
Hmm, looks like an issue with PHPCS now, don't think it's related to my code but I'll take a look later.

**Comment by erikyo** [comment]
I think we need to add 
`excludes: 'tests/'`

into actions WPCS action template. (check [here](https://github.com/erikyo/cf7-antispam/blob/main/.github/workflows/wpcs.yml))

**Comment by Sephsekla** [comment]
> I think we need to add `excludes: 'tests/'`
> 
> into actions WPCS action template. (check [here](https://github.com/erikyo/cf7-antispam/blob/main/.github/workflows/wpcs.yml))

Hmm, weird that this test passed on the previous commit though.

**Comment by erikyo** [comment]
indeed i was wrong, it is not an issue of with excluded paths and looking closely even files inside "./includes/" have the same problem. 

if you have the time try [updating wpcs](https://github.com/10up/wpcs-action/tree/stable), the one we are using is an old version

**Review by erikyo — APPROVED** [review]

**Pull request merged**